src/network.rs

```diff
@@ -35,7 +35,7 @@ pub enum IoEvent {
   SetTracksToTable(Vec<FullTrack>),
   GetMadeForYouPlaylistTracks(String, u32),
   GetPlaylistTracks(String, u32),
-  GetCurrentSavedTracks(Option<u32>, bool),
```

Comment:
Good work removing `bool` param, I should not have added it in the first place 😉